.travis.yml

```diff
@@ -3,13 +3,21 @@ language: go
 matrix:
   include:
     - go: 1.8.x
-      script: go test -v -race ./...
-    - go: 1.x
+      script:
+        - go test -v -race ./...
+    - go: 1.9.x
+      script:
+        - go test -v -race ./...
+    - go: 1.10.x
+      script:
+        - go test -v -race ./...
+    - go: 1.11.x
       script:
         - diff -u <(echo -n) <(gofmt -d .)
```

Comment:
Just curious.  What does this do?  `gofmt -d` displays the diff.  comparing the diff output to ??

Reply:
I copied this line a long time ago from elsewhere. I believe the principle is that if the developer forgot to run `gofmt`, this would produce a non-empty output, thus failing the Travis script.

Reply:
Hah.  Makes sense now.  It seems like `gofmt -d` will always have exit 0 and hence to check if there is a delta or not, compare it's output against echo -n which does not produce anything.  Neat.

Reply:
> I copied this line a long time ago from elsewhere.

#12. :)